src/apps/dashboard/dashboard-notification/dashboard-notification.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { FC, useCallback, MouseEvent } from "react";
+import Arrow from "../../../components/atoms/icons/arrow/arrow";
+import { Link } from "../../../components/atoms/link";
+
+interface DashboardNotificationProps {
+  notificationNumber: number;
+  notificationText: string;
+  notificationColor: string;
+  notificationLink: URL;
+  notificationClickEvent?: (modalId: string) => void | null;
+  notificationClickEventParam?: string;
+}
+
+const DashboardNotification: FC<DashboardNotificationProps> = ({
+  notificationNumber,
+  notificationText,
+  notificationColor,
+  notificationLink,
+  notificationClickEvent,
+  notificationClickEventParam
+}) => {
+  function stopPropagationFunction(e: MouseEvent) {
+    e.stopPropagation();
+  }
+  const notificationClickEventHandler = useCallback(
+    (e: MouseEvent) => {
+      stopPropagationFunction(e);
+      if (notificationClickEvent && notificationClickEventParam) {
+        notificationClickEvent(notificationClickEventParam);
+      }
+    },
+    [notificationClickEvent, notificationClickEventParam]
+  );
+  return (
```

Comment:
not quite sure about the design, but I think the header should be clickable instead of the element?